include/fmt/color.h

```diff
@@ -563,6 +563,52 @@ inline std::basic_string<Char> format(const text_style& ts, const S& format_str,
                  fmt::make_args_checked<Args...>(format_str, args...));
 }
 
+/** Formats a string with the given text_style and writes the output to ``out``.
+ */
+// GCC 8 and earlier cannot handle std::back_insert_iterator<Container> with
+// vformat_to<ArgFormatter>(...) overload, so SFINAE on iterator type instead.
+template <typename OutputIt, typename S, typename Char = char_t<S>,
+          FMT_ENABLE_IF(detail::is_output_iterator<OutputIt>::value)>
+OutputIt vformat_to(
+    OutputIt out, const text_style& ts, const S& format_str,
```

Comment:
S should be replaced with basic_string_view

Reply:
S can't be replaced because `basic_string_view` is missing an implicit constructor for a fixed size array like `const char(&)[5]`

Reply:
It doesn't matter. `format_to` should convert `S` into `basic_string_view` when calling this function.

Reply:
It seems like `to_string_view` can convert more types than the `basic_string_view` constructor.
In that case the `basic_string_view` constructor needs to be aligned with `to_string_view` or we need to rely on `to_string_view` to convert it for us. Replacing `const S&` with `basic_string_view` will lead to build failures otherwise.

Reply:
We should use `to_string_view`.